stories/toolbar/toolbar.stories.js

```diff
@@ -78,13 +78,13 @@ export const overflow = () => `<div style="height:250px">
             <div class="fd-popover__body fd-popover__body--right fd-popover__body--no-arrow"
                 aria-hidden="false"
                 id="wgxzK85912">
-                <div class="fd-toolbar__overflow__body">
-                    <button class="fd-button fd-button--transparent">Edit</button>
+                <div class="fd-toolbar__overflow">
```

Comment:
fd-toolbar__overflow-body

Reply:
Keeping fd-toolbar__overflow class instead of fd-toolbar__overflow-body as we are not using fd-toolbar__overflow for popover.